controllers/users.js

```diff
@@ -142,12 +142,26 @@ const postUserPicture = async (req, res) => {
     return res.boom.badImplementation('An internal server error occurred')
   }
 }
-
+const identityURL = async (req, res) => {
+  try {
+    const userId = req.userData.id
+    await userQuery.addOrUpdate(req.body, userId)
```

Comment:
What kind of validations will be done for URL?
Correct me if I am missing something here and sharing schema/ structure here would be helpful 👍🏻

Reply:
{
   identityURL: "https://abc.com"
}
This is req.body